sbin/common/config_init.sh

```diff
@@ -558,6 +562,9 @@ function configDefaults() {
 
   BUILD_CONFIG[ENABLE_SBOM_STRACE]="false"
 
+  # Set default value to "false", for maximum user convenience. "false" enables potentially-insecure functionality, like the dynamic download of boot JDKs.
```

Comment:
The boot jdk is one of few downloads, which you must manually force to happen.  Eg. alsa is on contrary download without warning.  Free type is downloaded as sources?  What else is/can be  downloaded?  /me just curious.  Thanx for the downlaod unification effort. Highly appreciated!

Reply:
Mainly boot JDKs. AIX, Alpine Linux, and Mac download boot JDKs if they can't find one locally.

There are also 22 other uses of wget, and many more that are indirect.

e.g. buildDocker.sh and dockerfile-generator wget many things, like the list of available releases and the mkdocker.sh script from openj9.